src/libtrx/game/gun/control.c

```diff
@@ -157,6 +157,9 @@ static void M_DecideRequestedWeapon(void)
     LARA_INFO *const lara = Lara_GetLaraInfo();
     const ITEM *const lara_item = Lara_GetItem();
     if (g_Input.draw) {
+        if (lara->last_gun_type == LGT_UNARMED) {
```

Comment:
Might be worth adding an inventory check that Lara has pistols. On this build you can do the following:

- `/play 13`
- `/fly`
- press draw input
- exit fly cheat
- Lara now has pistols in her holsters, but not inventory

Similar in HSH and when you then pickup the shotgun, pressing draw has no effect.

Reply:
>Similar in HSH and when you then pickup the shotgun, pressing draw has no effect.

Is this an OG behavior or TRX bug?

Reply:
TRX, but only this build - develop is OK.

Reply:
Fixed (though unsure whether the changelog needs an update)